### PR TITLE
made fix to requirements.txt

### PR DESCRIPTION
**Pull request by puneith**
@elmer-garduno Movielens fix in package and requirements.txt import

### CODE REVIEW

File: movielens/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
-tensorflow==1.2.0
-tensorflow-transform==0.1.10
+--index-url https://pypi.python.org/simple/
+
+-e .
```
Comment (elmer-garduno) [inline movielens/requirements.txt]: @puneith Not sure what this means? is it possible to add a comment on the requirements file?

File: movielens/setup.py [hunk 2]
```diff
@@ -16,8 +16,10 @@
 
 NAME = 'movielens'
 VERSION = '1.0'
-TENSORFLOW_TRANSFORM = 'tensorflow-transform==0.1.10'
+SIX='six==1.10.0'
+TENSORFLOW='tensorflow==1.4.0rc'
```
Comment (elmer-garduno) [inline movielens/setup.py]: Why 1.4.0rc, I'd rather set to 1.3.0 since we are sure that's compatible with tensorflow-transform==0.3.1

Comment (elmer-garduno) [inline movielens/setup.py]: Also as we have discussed on other forums having TF specified on the setup file can harm the dataflow performance by downgrading the optimized binary that's already present on the runners.

File: movielens/requirements.txt (hunk 1 above)
Comment (puneith) [reply movielens/requirements.txt]: Ok added

File: movielens/setup.py (hunk 2 above)
Comment (puneith) [reply movielens/setup.py]: Ok checked 1.3 works and updated.

Comment (puneith) [reply movielens/setup.py]: So what version is currently running with TFT 0.1.10? Oh its 1.3 as per the table in TFT right?

Comment (elmer-garduno) [reply movielens/setup.py]: Right now is 1.3, so I guess, we'll be fine. Thanks.

**Pull request merged**